### PR TITLE
default to theforeman.foreman.foreman callback on all Debians

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
includes #799 because I am lazy

File: manifests/plugin/ansible/params.pp [hunk 1]
```diff
@@ -18,11 +18,7 @@
       $runner_package_name = 'ansible-runner'
     }
     'Debian': {
-      if ($facts['os']['name'] == 'Ubuntu' and versioncmp($facts['os']['release']['major'], '20.04') >= 0) {
-        $callback = 'theforeman.foreman.foreman'
-      } else {
-        $callback = 'foreman'
-      }
+      $callback = 'theforeman.foreman.foreman'
```
Comment (evgeni) [inline manifests/plugin/ansible/params.pp:21]: today nothing installs the collection package on Debian -- should we do that in the style of `if $callback == 't.f.f' { ensure_package('ansible-collection-theforeman-foreman') }`? It's been packaged for debian for a while now

**Review by ekohl — APPROVED** [review]

File: manifests/plugin/ansible/params.pp (hunk 1 above)
Comment (evgeni) [reply manifests/plugin/ansible/params.pp:21]: Yes, I also thought about that, but we list FreeBSD in the supported OS'es, and I have no idea how things look there.

Comment (ekohl) [reply manifests/plugin/ansible/params.pp:21]: I'm not sure they package any plugin.

Comment (evgeni) [reply manifests/plugin/ansible/params.pp:21]: well, the one part is in https://github.com/theforeman/puppet-foreman_proxy/pull/801

the rest is up for future us.

Comment (evgeni) [reply manifests/plugin/ansible/params.pp:21]: https://www.freshports.org/sysutils/ansible/ is the "AIO" ansible, so it does actually contain our callback (even if in an older version).

there is also https://www.freshports.org/sysutils/py-ansible-runner/, so I might end up in another rabbit hole.

**Pull request merged**